content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -117,10 +117,11 @@ Enable the Datadog profiler. Added in version `0.69.0`. See [Enabling the PHP Pr
 **Default**: `1`<br>
 Whether to enable the endpoint data collection in profiles. Added in version `0.79.0`.
 
-`DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED`
-: **INI**: `datadog.profiling.experimental_allocation_enabled`. INI available since `0.84.0`.<br>
-**Default**: `0`<br>
-Enable the experimental allocation size and allocation bytes profile type. Added in version `0.84.0`.
+`DD_PROFILING_ALLOCATION_ENABLED`
+: **INI**: `datadog.profiling.allocation_enabled`. INI available since `0.88.0`.<br>
+**Default**: `1`<br>
+Enable the allocation size and allocation bytes profile type. Added in version `0.88.0`.<br>
+**Note:** Supersedes `DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED` environment variable (`datadog.profiling.experimental_allocation_enabled` INI setting) which was available since `0.84`. If both are set this one takes precedence.
```

Comment:
```suggestion
**Note**: This supersedes the `DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED` environment variable (`datadog.profiling.experimental_allocation_enabled` INI setting), which was available since `0.84`. If both are set, this one takes precedence.
```